packages/preferences/src/components/preferences-modal-tabs/style.scss

```diff
@@ -1,12 +1,15 @@
 $vertical-tabs-width: 160px;
 
 .preferences__tabs-tablist {
-	position: absolute;
+	position: absolute !important;
 	top: $header-height + $grid-unit-30;
 	// Aligns button text instead of button box.
 	left: $grid-unit-20;
 	width: $vertical-tabs-width;
 
+	&::after {
```

Comment:
Is this something we need to handle in the component itself? Perhaps if we provide an API to declare the indicator to be optional?

Reply:
This seems like an escape hatch and it's only used here. You could also say the same about everything else that's being customized or overriden here (a lot). I would advise against supporting this to prevent configuration-creep from building up. If this pattern becomes "official" or, at least, common enough, then I would agree.

At the root of this I think the problem is that `Tabs` from `@wordpress/components` is not what was needed here. Rather, it seems like we need tabs that look a certain way, and for that it seems like using Ariakit with the styles wanted for this interface is the way to go.